asm/macros/battle_script.inc

```diff
@@ -2005,6 +2005,19 @@
 	.byte \stat
 	.endm
 
+	.macro trywindriderpower battler:req, ptr:req
+	various \battler, VARIOUS_TRY_WIND_RIDER_POWER
+	.4byte \ptr
+	.endm
+
+	.macro activateweatherabilities battler:req
```

Comment:
I would rename to `activateweatherchangeabilities` or just `weatherchangeabilities`